lib/tools/ts-inspector.js

```diff
@@ -121,21 +121,32 @@ var nextPesPacket_ = function(bytes, pmt, startIndex, type) {
 var getFullAdtsHeader_ = function(bytes, pmt, packet, startIndex, endIndex) {
   var
     headerOffset,
-    headerBytes,
     i,
     crcPresent,
     headerLength,
     nextPacketResult,
+    // elementary stream bytes
+    esBytes,
+    packetEsBytes,
+    pesHeaderLength,
+    headerBytes,
+    // to get the elementary stream bytes from the packet, we add 4 for the TS header,
+    // then since we know it is an audio PES packet, we can bypass the 4 byte start code,
+    // 2 byte PES packet length, and the first 2 optional PES header bytes to get to the
+    // PES header length (remainder) byte
+    pesHeaderLengthIndex = 4 + 4 + 2 + 2,
```

Comment:
This doesn't take into account the adaption field of the TS packet header. https://github.com/videojs/mux.js/blob/master/lib/m2ts/m2ts.js#L222